public/index.html

```diff
@@ -0,0 +1,370 @@
+        <progress value="91" max="100"></progress>
+        <p class="skill-list">ILLUSTRATOR</p>
+        <progress value="63" max="100"></progress>
+        <p class="skill-list">PHP</p>
+        <progress value="55" max="100"></progress>
+        <p class="skill-list">WORDPRESS</p>
+        <progress value="97" max="100"></progress>
+        <p class="skills-para">Do you want be one of Avengers? Join us!</p>
+        <button class="button-contact" type="submit">CONTACT US</button>
+    </section>
+    <section class="section-white">
+        <object class="clip" type="image/svg+xml" data="./imgs/clipblack.svg"></object>
+        <a id="team"></a>
+        <h1>TEAM</h1>
+        <p>Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+        <ul class="flex-container">
+            <li class="flex-item-4">
+                <h2>JONATHAN DOES</h2>
+                <h3 class="team-title">CEO, UI&#38;UX designer</h3>
+                <img class="face-pic" src="./imgs/jondoe.png" alt="head picture of Jonathon Doe">
+                <ul class="social-media">
+                    <li><a href=""><span class="socicon">b</span></a></li>
+                    <li><a href=""><span class="socicon">a</span></a></li>
+                    <li><a href=""><span class="socicon">D</span></a></li>
+                    <li><a href=""><span class="socicon">j</span></a></li>
+                    <li><a href=""><span class="socicon">c</span></a></li>
+                </ul>
+                <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+            </li>
+            <li class="flex-item-4">
+                <h2>JANNETE DOE</h2>
+                <h3 class="team-title">Bussines &#38; marketing</h3>
+                <img class="face-pic" src="./imgs/jandoe.png" alt="head picture of Jannete Doe">
+                <ul class="social-media">
+                    <li><a href=""><span class="socicon">b</span></a></li>
+                    <li><a href=""><span class="socicon">a</span></a></li>
+                    <li><a href=""><span class="socicon">D</span></a></li>
+                    <li><a href=""><span class="socicon">j</span></a></li>
+                    <li><a href=""><span class="socicon">c</span></a></li>
+                </ul>
+                <p class="team-para">Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+            </li>
+            <li>
```

Comment:
What's this doing here? You have an extra—unclosed—`li`. This will fail a validator. Markup errors like this can be very hard to debug later.